internal/jobcontainers/logon.go

```diff
@@ -31,7 +31,9 @@ func groupExists(groupName string) bool {
 	); err != nil {
 		return false
 	}
-	defer windows.NetApiBufferFree(p)
+	defer func() {
+		_ = windows.NetApiBufferFree(p)
+	}()
```

Comment:
Curious why our linter didn't catch this on the pr..

Reply:
But it does after check in

Reply:
Regardless, thank you!

Reply:
yeah, not sure. for some reason the branch build fails in my fork, but not on PR

Reply:
Don't like not understanding why our linter does what it does.. -_-